types/wordpress__editor/store/selectors.d.ts

```diff
@@ -56,6 +57,8 @@ export {
     isValidTemplate,
 } from '@wordpress/block-editor/store/selectors';
 
+type StateType = Record<string, unknown>;
```

Comment:
Not super important to have this typed, since the public API for the state is selectors and actions.